coffee_lib/src/macros.rs

```diff
@@ -48,5 +48,18 @@ macro_rules! sh {
     };
 }
 
-pub use error;
-pub use sh;
+/// handle_httpd_response macro is the macro that handles HTTPD responses
+#[macro_export]
+macro_rules! handle_httpd_response {
+    ($result:expr, $msg:expr) => {
+        match $result {
+            Ok(_) => Ok(HttpResponse::Ok().body(format!($msg))),
+            Err(err) => Err(actix_web::error::ErrorInternalServerError(format!(
+                "Error: {}",
+                err
+            ))),
+        }
+    };
+}
```

Comment:
no this should stay in the httpd, we do not have access to the HTTP Response here